src/borg/helpers/parseformat.py

```diff
@@ -377,6 +378,14 @@ class Location:
         |                                                   # or
         """ + optional_archive_re, re.VERBOSE)              # archive name (optional, may be empty)
 
+    win_file_re = re.compile(r"""
+        (?:file:\/\/)?                                      # optional file protocol
+        (?P<path>
+            (?:[a-zA-Z]:\\)?                                # Drive letter followed by a backslash (optional)
```

Comment:
drive letter followed by **a colon and** a backslash

Reply:
hmm, this might create strange mixups like file://c:\foobar - shouldn't we just forbid using backslashes?

Reply:
No, windows uses backslashes as path separator and forbiding them would make it harder to use borg in scripts and on the command line.

Reply:
i just tried on windows 10, invoking something like `notepad c:/windows/win.ini` works.

we could just convert backslashes to slashes before doing anything else (if just forbidding them is too much trouble).

Reply:
I know, forward slashes are supported, but any windows command or environment variable will returns paths with backslashes.
For example, this would fail if backslashes are not allowed:
````
borg create ::b-1 %HOMEDRIVE%%HOMEPATH%
````

It would be possible to replace backslashes with slashes early in the process (before parsing) but I don't see how this would simplify the parsing other than the regex would not require backslashes? We still need to distinguish between Windows and non Windows because of the drive letter.